pkg/activator/net/throttler.go

```diff
@@ -314,8 +315,14 @@ func NewThrottler(ctx context.Context,
 	return t
 }
 
-// Run starts the throttler and blocks until updateCh is closed.
-func (t *Throttler) Run(updateCh <-chan revisionDestsUpdate) {
+// Run starts the throttler and blocks until the context is done.
+func (t *Throttler) Run(ctx context.Context, rt http.RoundTripper) {
```

Comment:
Do we need to pass the roundtripper here? Doesn't seem to be needed for tests at least? Dropping it would resolve the weirdness a bit.